autotests/testkeyverification.cpp

```diff
@@ -15,23 +17,37 @@ private Q_SLOTS:
     void testVerification()
     {
         CREATE_CONNECTION(a, "alice1", "secret", "AliceDesktop")
-        CREATE_CONNECTION(b, "alice1", "secret", "AlicePhone")
+        CREATE_CONNECTION(b, "alice2", "secret", "AlicePhone")
+
+        a->requestDirectChat(b->userId());
+            connectSingleShot(b.get(), &Connection::invitedRoom, this, [=](Quotient::Room* room, Quotient::Room*) {
```

Comment:
```suggestion
        connectSingleShot(b.get(), &Connection::invitedRoom, this, [=](Quotient::Room* room) {
```
You can omit trailing signal parameters you don't use in the slot.